cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -314,6 +322,17 @@ install() {
         _env_script_path="$HOME/.cargo/env"
         _install_dir_expr='$HOME/.cargo/bin'
         _env_script_path_expr='$HOME/.cargo/env'
+
+        # If we need to override the above
+        if [ -n "${CARGO_DIST_FORCE_INSTALL_DIR:-}" ] && [ "${CARGO_DIST_FORCE_INSTALL_DIR}" != "$HOME/.cargo" ]; then
+            _install_home="$CARGO_DIST_FORCE_INSTALL_DIR"
```

Comment:
....i can't find any code that reads _install_home... can we just delete it from everywhere, or do you want to keep this around for future work?

Reply:
It's funny, when I started I thought `$_install_home` was used to construct further paths, but it isn't. It's probably safe to remove it for now. It might get used at some point when we start doing more complex path structures (eg libs in `$_install_home/lib`), but we're not there yet so it can get removed until that happens. I'll do that in a followup commit.